diff --git a/types/extrinsic.d.ts b/types/extrinsic.d.ts
index 0ad479fe3dd04f2fbd8704fdeeab8dcfdf2daafd..c9ee33c6465911394eea413317dc7b1c1ac326b0 100644
--- a/types/extrinsic.d.ts
+++ b/types/extrinsic.d.ts
@@ -101,7 +101,7 @@ export interface SignerResult {
     /**
      * @description The id for this request
      */
-    id: number;
+    id: string;
     /**
      * @description The resulting signature in hex
      */
